install_scripts/setup_sadm.sh

```diff
@@ -12,6 +12,27 @@ Server = https://repo.prologin.org/
 EOF
 fi
 
+sed -i 's/^[#]IgnorePkg.*$/# Ignoring all of the following packages\
+# Install utilities\
+IgnorePkg = arch-install-scripts syslinux mkinitcpio-nfs-utils\
+# Boot and file systems utilities\
+IgnorePkg = nbd tftp-hpa conntrack-tools openresty ipxe-sadm-git nfs-utils\
+# Network\
+IgnorePkg = openssh dhcp bind ipset dnsutils rsync tcpdump\
+# Useful tools\
+IgnorePkg = vim rxvt-unicode-terminfo git strace wget ethtool tree mtr iperf\
+IgnorePkg = atop htop iotop iftop nethogs jq tmux bash-eternal-history\
+# Monitoring\
+IgnorePkg = grafana prometheus prometheus-node-exporter-git\
+# Web\
+IgnorePkg = sqlite postgresql postgresql-libs\
+# Python\
+IgnorePkg = python python2 python-pip python-virtualenv\
+# Misc\
+IgnorePkg = sddm isolate-git prologin\/pamtester libyaml libxslt pwgen\
```

Comment:
I'm pretty sure we might wanna upgrade these at some point during an install

Reply:
Fixed according to what was said in #120